powershell/cmdlets/class.ts

```diff
@@ -1009,8 +1021,13 @@ export class CmdletClass extends Class {
             // no return type. Let's just return ... true?
             yield 'WriteObject(true);';
           });
-          yield 'var headers = Microsoft.Graph.PowerShell.ResponseHeader.Helper.ResponseHeaderHelper.ToResponseHeader(responseMessage);'
-          yield 'WriteObject(headers);'
+          yield `var headers = new System.Collections.Generic.Dictionary<string, System.Collections.Generic.IEnumerable<string>>();`
```

Comment:
It would be useful to extract this logic out into a function. The repetition increases the chances of bugs.